go/oasis-test-runner/scenario/e2e/stake_cli.go

```diff
@@ -569,6 +592,63 @@ func (sc *stakeCLIImpl) testAmendCommissionSchedule(childEnv *env.Env, cli *cli.
 	return nil
 }
 
+// testAllowWithdraw tests setting an allowance and withdrawing.
+func (sc *stakeCLIImpl) testAllowWithdraw(childEnv *env.Env, cli *cli.Helpers, src, beneficiary api.Address, beneficiaryEntityDir string) error {
+	var srcNonce uint64 = 5
```

Comment:
It would be nice to not hard-code the nonces. Something like:

```suggestion
	srcNonce, err := sc.getAccountNonce(childEnv, src)
	if err != nil {
		return fmt.Errorf("getAccountNonce for source account %s: %w", src, err)
	}
	beneficiaryNonce, err := sc.getAccountNonce(childEnv, beneficiary)
	if err != nil {
		return fmt.Errorf("getAccountNonce for beneficiary account %s: %w", beneficiary, err)
	}
```

Reply:
Oh I guess this changed while rebasing on master as previousl there was no such thing :-)

Reply:
He he, indeed, I added this in #3559 :slightly_smiling_face:.